httpapi_plugins/ftd.py

```diff
@@ -64,7 +64,8 @@
 
 BASE_HEADERS = {
     'Content-Type': 'application/json',
-    'Accept': 'application/json'
+    'Accept': 'application/json',
+    'User-Agent': 'FTD Ansible'
```

Comment:
is there any sense add package version and/or OS name, version to header value?

Reply:
Makes sense, I'll add Ansible version, so it will be `FTD Ansible/2.7.0` or similar. OS name might not be very relevant as we just want to identify requests coming from Ansible.